resources/docs/simulation/concepts/design-considerations/README.md

```diff
@@ -78,7 +78,7 @@ A process for handling conflicts must be defined in any modeling approach - in t
 To streamline this, consider:
 
 * including any information in messages which might help resolve the dispute - e.g. include a second preference order if the first cannot be fulfilled \(where the agent receiving the order can handle both\).
-* for more complex scenarios, using a mediating or manager agent which receives information from multiple agents, and can determine the best match in a single step. 
+* for more complex scenarios, using a mediating or manager agent which receives information from multiple agents, and can determine the best match in a single step.
```

Comment:
Prettier would normally change this to `-`. Might be worth introducing it too to avoid ‘breaking’ `git blame` again in the future.

Reply:
@kachkaev tbh I'm not sure which part you're referring to here. The commit you're referring to is removing a trailing space from the sentence.

Reply:
We are currently using markdownlint without prettier and can make our files even cleaner if we combine these tools. Prettier will replace `*` with `-` when we add it. If we do this shortly after merging this PR, our `git blame` won't be polluted by non-essential changes later.